app/app.go

```diff
@@ -285,6 +291,12 @@ func download(c *cli.Context, videoURL string) error {
 		return err
 	}
 
+	if c.Bool("rename") {
+		for _, v := range data {
+			v.Title += fmt.Sprintf("-%v", time.Now().Unix())
```

Comment:
If more than one file is downloaded at a time, the timestamp is the same inside this loop